packages/bpk-component-table/src/BpkTableHead-test.js

```diff
@@ -26,7 +26,7 @@ describe('BpkTableHead', () => {
     const tree = renderer
       .create(
         <BpkTableHead>
-          <th />
+          <th>Skyscanner</th>
```

Comment:
Was this deliberate?

Reply:
Yeah, it was to solve this error:

```
error  A control must be associated with a text label  jsx-a11y/control-has-associated-label
```

Reply:
Seems strange - what control is it referring to?

Reply:
The `th` element — it makes sense, because `<th />` evaluates to `<th></th>` which is invalid, it needs to have content. So it's just enforcing that. You'll see a similar update in this PR on a `<Button />`.

Reply:
I'm an idiot. I thought you were removing the text, but you're adding it which totally makes sense 👍

Reply:
Haha no worries